routes/keyword.js

```diff
@@ -7,19 +7,41 @@ const Keyword = require('../models/KeywordModel.js')
 
 const router = express.Router()
 
-router.get('/:word', (req, res) => {
-    console.log(req.params)
+router.get('/word/:word', (req, res) => {

```

Comment:
This route is already mounted at `/keywords`, so to access this function you would need to `GET /keywords/:word`